src/components/ProductList.tsx

```diff
@@ -22,10 +31,13 @@ export interface IProps {
 const ProductList: React.FC<IProps> = ({ content, category }) => {
   const shadow = useColorModeValue("tableBox.light", "tableBox.dark")
 
+  const ariaLabelledby = "cat-name"
```

Comment:
I just had a question abut this label. Im a little confused what "cat-name" is. Should the be "category-name", or should we actually be passing the category prop instead?

Reply:
This is just an id because it is being used in two places (lines 40 and 51) for the `aria-labelledby` feature.

I can be clearer here in using a more verbose id, along with uppercase variable naming to denote this as a const variable 😁

Reply:
```suggestion
  const ariaLabelledby = "CATEGORY-NAME"
```

Sure, lets go with this :)

Reply:
Sure! I pushed those changes up. This should be good to go now :)